R/service-azure_openai.R

```diff
@@ -110,31 +110,33 @@ query_api_azure_openai <-
 retrieve_azure_token <- function() {
   rlang::check_installed("AzureRMR")
```

Comment:
swap out for Microsoft365R?

Reply:
Good catch, I replaced with AzureGraph, as addressed below.